pkg/builder/TestTwoSampleTTestBuilder_test.go

```diff
@@ -443,3 +443,49 @@ func TestTwoSampleTTestBuilder_test__match_exp_short_1(t *testing.T) {
 		t.Fatal("test_1 wrong value :", cellPtr.value)
 	}
 }
+
+// this test has inputs that should return a value of 0 exp missing all data)
+func TestTwoSampleTTestBuilder_test__missing_one_population(t *testing.T) {
+	err := cellPtr.SetGoodnessPolarity(gp)
+	if err != nil {
+		t.Fatal(fmt.Sprint("TestTwoSampleTTestBuilder_test__missing_one_population - SetGoodnessPolarity - error message : ", err))
+	}
+	err = cellPtr.SetMinorThreshold(minorThreshold)
+	if err != nil {
+		t.Fatal(fmt.Sprint("TestTwoSampleTTestBuilder_test__missing_one_population - SetMinorThreshold - error message : ", err))
+	}
+	err = cellPtr.SetMajorThreshold(majorThreshold)
+	if err != nil {
+		t.Fatal(fmt.Sprint("TestTwoSampleTTestBuilder_test__missing_one_population - SetMajorThreshold - error message : ", err))
+	}
+	if err != nil {
+		t.Fatal(fmt.Sprint("TestTwoSampleTTestBuilder_test__missing_one_population - SetInputData - error message : ", err))
+	}
+	epoch := time.Now().Unix()
```

Comment:
I think it'd be better to have a static time in our unit tests - that way we're sure we're always testing the same thing and don't have to worry about flaky tests.

```suggestion
	epoch := 1682113500 // Unix epoch in seconds - Friday, April 21, 2023 9:45:00 PM
```

Reply:
done